<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Euro数据集制作工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>视频分割</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>视频路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>图片输出路径</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="videoIputPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>F:/ffmpeg-2023-05-25-git-944243477b-essentials_build/bin/1.avi</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vidoe2imgoutputPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/图片资料</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startCutVideo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始视频分割</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>命名规则</string>
     </property>
    </widget>
    <widget class="QComboBox" name="changeType1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Euro数据集</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="videoIputPath_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vidoe2imgoutputPath_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
    <widget class="QComboBox" name="videoType">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>mp4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>avi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wmv</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>视频格式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ffmpeg路径</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ffmpegPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>F:/ffmpeg-2023-05-25-git-944243477b-essentials_build/bin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ffmpegPath_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>切割帧率</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cuteFPS">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>35</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>图片数据集转格式</string>
    </property>
    <widget class="QPushButton" name="startChange">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始转格式</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="imgOutputPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>图像输入目录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>图像输出目录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="imgIputPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="changeType2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Euro数据集</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>命名规则</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imgIputPath_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imgOutputPath_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
